packages/ocamleditor/ocamleditor.1.13.4~preview.4.10/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+name: "ocamleditor"
```

Comment:
```suggestion
```

This field is redundant. Opam will override it with what it infers from the path and may give rise to errors